streaming/index.js

```diff
@@ -372,6 +355,12 @@ if (cluster.isMaster) {
     const token    = location.query.access_token;
     const req      = { requestId: uuid.v4() };
 
+    ws.isAlive = true;
```

Comment:
According to example: <https://github.com/websockets/ws#how-to-detect-and-close-broken-connections>